### DISCUSSION TITLE
goroutine · GitBook

### DISCUSSION THREAD

**Discussion by xzghua** [wzcu/interview-comment#158] [Comments]
http://interview.wzcu.com/Golang/goroutine.html

**Comment by the-stardust** [comment]
真的强

**Comment by xzghua** [comment]
> @the-stardust
> 真的强

哈哈 看懂了没?   我看了不少文章 才总结出来的!  但是还是有地方不太懂..

**Comment by the-stardust** [comment]
> @xzghua
> > @the-stardust
> > 真的强
> 
> 哈哈 看懂了没?   我看了不少文章 才总结出来的!  但是还是有地方不太懂..   

看来一遍 一知半解 还要花几天时间细品下哈哈哈

**Comment by zzzzz1797** [comment]
大佬、大佬。学习了
